jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/BottomNavigationTests.java

```diff
@@ -47,19 +47,19 @@ public void hideOnScrollTest() {
     @Test
     public void shiftTest() {
         bottomNavigationShift.get(1).has().text("Video");
-        bottomNavigationShift.select(1);
+        bottomNavigationShift.selectByText("Video");
```

Comment:
you can specify collection of test data and iterate through it to get short and clear test which can simply supported in the future.